EnhancedMediaPlayer/MediaPlayerTappableView.swift

```diff
@@ -0,0 +1,132 @@
+// MediaPlayerTappableView.swift
+
+import SwiftUI
+
+struct MediaPlayerTappableView: View {
+    @ObservedObject var viewModel: ViewModel
+
+    var body: some View {
+        ZStack {
+            renderSeekLayer()
+            renderControlsLayer()
+        }
+    }
+
+    @ViewBuilder func renderControlsLayer() -> some View {
+        Rectangle()
+            .foregroundColor(.clear)
+            .contentShape(Rectangle())
+            .frame(width: Constants.tappableControlScreenWidth, height: .infinity)
+            .onTapGesture {
+                viewModel.onTapControlsArea?()
+            }
+    }
+
+    @ViewBuilder func renderSeekLayer() -> some View {
+        HStack(spacing: .zero) {
+            renderTappableScreen(.backward)
+            renderTappableScreen(.forward)
+        }
+    }
+
+    @ViewBuilder private func renderTappableScreen(_ seek: Seek) -> some View {
+        ZStack {
+            renderSeekTapsDetector(seek)
+            renderSeekIcon(seek)
+        }
+    }
+
+    @ViewBuilder func renderSeekTapsDetector(_ seek: Seek) -> some View {
+        Rectangle()
+            .foregroundColor(.clear)
+            .contentShape(Rectangle())
+            .onTapGesture {
+                viewModel.onTapSeekArea?(seek)
+            }
+    }
+
+    @ViewBuilder func renderSeekIcon(_ seek: Seek) -> some View {
+        if viewModel.visibleSeek == seek {
+            seek.systemImage
+                .resizable()
+                .frame(width: Constants.iconSize.width, height: Constants.iconSize.height)
+                .foregroundColor(.white)
+        }
+    }
+}
+
+extension MediaPlayerTappableView {
+    class ViewModel: ObservableObject {
+        @Binding var seekState: SeekState
+        @Binding var isSeekByTapEnabled: Bool
+        @Binding var visibleSeek: Seek?
```

Comment:
None of these need to be `@Bindings`. They can all be regular `let` variables, `@Bindings` should be used when you want to change a variable that was receive on `init` and affect the variable that was passed on the `init`